dv/uvm/core_ibex/directed_tests/directed_testlist.yaml

```diff
@@ -0,0 +1,25 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+##########################################################
+
+# This file largely copies the formatting of the testlist.yaml
+# used by riscv-dv, but only specifies directed tests.
+
+
+##########################################################
```

Comment:
Shouldn't the 'access_pmp_overlap.S' test be in here?

Reply:
thanks for pointing out. Added it here and modified it with a handler to cleanly fail in case of unexpected exception.